.github/workflows/builds.yml

```diff
@@ -136,13 +136,11 @@ jobs:
       - name: Use Node.js specified in the '.nvmrc' file
         uses: actions/setup-node@v3
         with:
-          node-version-file: '.nvmrc'
+          node-version-file: ".nvmrc"
       - name: Install npm
-        run: npm i -gf npm@8.7.0
+        run: npm i -gf "npm@$(jq --raw-output .engines.npm <"package.json")" && npm -v
```

Comment:
Is `jq` present on the CI agents?

Reply:
It is :
- linux : https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-Readme.md#tools
- mac : https://github.com/actions/virtual-environments/blob/main/images/macos/macos-12-Readme.md#utilities
- windows : https://github.com/actions/virtual-environments/blob/main/images/win/Windows2022-Readme.md#tools (but I don't know the CLI syntax on windows to get the result from a subcommand)

Reply:
Could you please try it out? Or at least add a `TODO` comment so that someone can take a look at it in the future?

Reply:
Added the command for Windows as well (fortunately it runs on Powershell)